great_expectations/expectations/metrics/query_metrics/query_column.py

```diff
@@ -65,7 +65,7 @@ def _sqlalchemy(
         engine: sqlalchemy_engine_Engine = execution_engine.engine
         result: List[sqlalchemy_engine_Row] = engine.execute(sa.text(query)).fetchall()
 
-        return [dict(element) for element in result]
+        return [element._asdict() for element in result]
```

Comment:
❤️ -- great thing (I had already done this for Query metrics).  Please note that same can be applied to `Spark`.  Thank you.